src/localizer/CameraLocalizerPluginFactory.cpp

```diff
@@ -985,6 +993,15 @@ void CameraLocalizerPluginFactory::describeInContext(OFX::ImageEffectDescriptor&
       param->setHint("Clear all output values and cache");
       param->setEnabled(true);
       param->setParent(*groupOutput);
+      param->setLayoutHint(OFX::eLayoutHintDivider);
+    }
+
+    {
+      OFX::PushButtonParamDescriptor *param = desc.definePushButtonParam(kParamCreateScene);
+      param->setLabel("Create 3D Scene");
+      param->setHint("Build the 3D scene from data");
+      param->setEnabled(true);
```

Comment:
As this is only valid when you have a python script.
We can declare it secret & disabled and change this status from script. So if you don't have the script, you don't have the option.